addons/mail/static/src/widgets/form_renderer/form_renderer_tests.js

```diff
@@ -448,6 +457,204 @@ QUnit.test('chatter should become enabled when creation done', async function (a
+    assert.doesNotHaveClass(formViewEl, 'o_xxl_form_view',
+        "Form view is not xxl"
+    );
+
+    assert.strictEqual(
+        document.querySelector('.breadcrumb-item.active').textContent,
+        'second partner',
+        "'second partner' form view is opened"
+    );
+    assert.strictEqual(controllerContentEl.scrollTop, 0,
+        "The initial scrollTop of the form view is 0"
+    );
+    // scroll message list to a big value to get to the bottom
+    let scrollRecord1;
+    await afterNextRender(() => {
+        controllerContentEl.scrollTop = 9000;
+        scrollRecord1 = controllerContentEl.scrollTop;
+        return triggerEvent(
+            document.querySelector('.o_ThreadViewer_messageList'),
+            'scroll'
+        );
+    });
+    assert.ok(scrollRecord1 > 0, "We have scrolled the record to a non-zero value");
+    await afterNextRender(() =>
+        document.querySelector('.o_pager_next').click()
+    );
+    assert.strictEqual(
+        document.querySelector('.breadcrumb-item.active').textContent,
+        'third partner',
+        "'third partner' form view is opened"
+    );
+    assert.strictEqual(controllerContentEl.scrollTop, 0,
+        "The scrollTop of the form view is still 0 on another record"
+    );
+    await afterNextRender(() =>
+        document.querySelector('.o_pager_previous').click()
+    );
+    assert.strictEqual(controllerContentEl.scrollTop, 0,
+        "The scrollTop of the form view is still 0 on first record"
+    );
+});
+
+QUnit.test('Form view with primary button not scrolled when posting message', async function (assert) {
+    assert.expect(7);
+
+    this.data['res.partner'].records = [{
+        activity_ids: [],
+        id: 2,
+        display_name: "second partner",
+        description: [...Array(60).keys()].join('\n'),
+        message_ids: [],
+        message_follower_ids: [],
+    }];
+
+    const messages = [...Array(60).keys()].map(id => {
+        return {
+            author_id: [10, "Demo User"],
+            body: `<p>Message ${id + 1}</p>`,
+            date: "2019-04-20 10:00:00",
+            id: id + 1,
+            message_type: 'comment',
+            model: 'res.partner',
+            record_name: 'General',
+            res_id: 2,
+        };
+    });
+    this.data['mail.message'].records = messages;
+
+    await this.createView({
+        data: this.data,
+        hasView: true,
+        // View params
+        View: FormView,
+        model: 'res.partner',
+        arch: `
+            <form string="Partners">
+                <header>
+                    <button name="primaryButton" string="Primary" type="object" class="oe_highlight" />
+                </header>
+                <sheet>
+                    <field name="name"/>
+                    <field name="description"/>
+                </sheet>
+                <div class="oe_chatter">
+                    <field name="message_ids" options="{'post_refresh': 'always'}"/>
```

Comment:
nope, otherwise I wouldn't have to put that here. If it is not present, the listener on o-message-posted is never added

Reply:
Ok my bad, we talked about this with @xdu-odoo and I had in my mind that we removed it already. Maybe he can tell us more, otherwise just ignore this suggestion.

Reply:
```
this.chatterFields = {
    (...)
    hasRecordReloadOnAttachmentsChanged: post_refresh === 'always',
    hasRecordReloadOnMessagePosted: !!post_refresh,
   (...)
};
```

This is then used to instanciate the chatter component, it's still supported